games/TicTacToe/README.md

```diff
@@ -30,28 +30,41 @@ The project consists of a smart contract.
 
 ```sh
 TicTacToe
-├── project
-│   ├── contracts
-│   │   └── tictactoe-contract
-│   │       ├── src/main.sw
-│   │       └── tests/harness.rs
-│   ├── README.md
-│   └── SPECIFICATION.md
-├── ui
-│   ├── README.md
-│   └── SPECIFICATION.md
+├── packages
+├────── project
+│   ├────── contracts
+│   │   └────── tictactoe-contract
+│   │       ├────── src/main.sw
+│   │       └────── tests/harness.rs
+│   ├────── README.md
+│   └────── SPECIFICATION.md
+├────── app
+│   ├────── package.json
+│   ├────── index.html
+│   └────── src/main.tsx
 └── README.md
 ```
 
 ## Running the project
 
 ### User interface
 
-TODO: The user interface does not currently exist therefore its [README.md](ui/README.md) and [SPECIFICATION.md](ui/SPECIFICATION.md) are empty.
+In order to run the subsequent commands change into the following directory `/path/to/TicTacToe/packages/app/<here>`.
+
+#### Run the frontend application
+
+```bash
+pnpm install
+pnpm build
+pnpm fuels dev
+pnpm dev
+```
+
+You will need to install the [Fuel wallet](https://wallet.fuel.network/docs/install/) in order to interact with the application. Now you can open your browser and interact with the tic-tac-toe contract through the frontend.
```

Comment:
Seems like you also need to modify the chainConfig to fund your wallet locally, right?

It would be nice to explain how to play a bit more, and that you have two accounts and connect them both